plugins/BEdita/Core/src/Model/Table/ApplicationsTable.php

```diff
@@ -164,7 +171,8 @@ protected function findApiKey(Query $query, array $options)
             ->where([
                 $this->aliasField('api_key') => $options['apiKey'],
                 $this->aliasField('enabled') => true,
-            ]);
+            ])
+            ->cache(sprintf('app_%s', $options['apiKey']), static::CACHE_CONFIG);
```

Comment:
Should we also invalidate cache in `afterSave()` and `afterDelete()`?

Reply:
That's right, I'll do that

Reply:
Done